ReactNativeClient/lib/models/Note.js

```diff
@@ -481,6 +481,25 @@ class Note extends BaseItem {
 		return note;
 	}
 
+	static async duplicateMultipleNotes(noteIds, options = null){
+		/*
+			if options.uniqueTitle is true, a unique title for the duplicated file will be assigned.
+		 */
+		const uniqueTitle = options && options.uniqueTitle;
+
+		for(const noteId of noteIds){
+			const noteOptions = {};
+
+			//If uniqueTitle is truthy, set the original note's name as root for the unique title.
+			if(uniqueTitle){
+				const originalNote = await Note.load(noteId);
+				noteOptions.uniqueTitle = originalNote.title;
+			}
+
+			await Note.duplicate(noteId, noteOptions);
+		}
+	}
```

Comment:
I like that you created a method for this as that will make it easier to re-use in other apps. The `uniqueTitle` option might be confusing since it has a different meaning than in the `duplicate()` method. Could you please rename it to something else that shows it's a boolean, maybe `ensureUniqueTitle`?

Reply:
Just changed!